Sources/Controllers/ALKConversationViewController.swift

```diff
@@ -410,6 +410,14 @@ open class ALKConversationViewController: ALKBaseViewController, Localizable {
         })
     }
 
+    func switchDynamicMode(isDynamic: Bool) {
+        if isDynamic {
+                UIApplication.shared.windows.first?.overrideUserInterfaceStyle = .unspecified
+        } else {
+                UIApplication.shared.windows.first?.overrideUserInterfaceStyle = .light
+        }
+    }
+    
     open func updateAssigneeOnlineStatus(userId: String){}
```

Comment:
its is used in different view controller and repeated. Can we add this into VC extension & use in all places ?

Reply:
moved it to UiViewContoller extension.